src/crypto/local.js

```diff
@@ -17,8 +17,14 @@
 /* exported getCurvePoints */
 /* exported getBigNumFromBytes */
 /* exported getActiveECSettings */
+/* exported verifyCommitments */
+
 "use strict";
 
+const parseKeys = require("parse-asn1");
```

Comment:
I don't know where is the right place to include these packages.

Reply:
Good point, it might be a good time to start a separate file for requirements that is then included in the building process?

Reply:
Added a requires.js file.